packages/playground/src/app.tsx

```diff
@@ -324,35 +324,35 @@ const themes: PlaygroundProps['themes'] = {
     theme: SuiTheme,
   },
   shadcn: {
-    stylesheet: 'https://cdn.jsdelivr.net/gh/tuanphung2308/rjsf-shadcn-css@main/public/default.css',
+    stylesheet: '//cdn.jsdelivr.net/gh/tuanphung2308/rjsf-shadcn-css@main/public/v4/default.css',
```

Comment:
Will these changes cause the look of the theme to change? If so, consider updating the `screenshot` of the theme... Oh wait, your `README.md` doesn't contain a screenshot. Can you add one?